### DISCUSSION TITLE
Make `zen.view.use-deprecated-urlbar` available in about:preferences

### DISCUSSION THREAD

**Discussion by Eragonfr** [zen-browser/desktop#3801] [Ideas]
Well everything is in the title.
`zen.view.use-deprecated-urlbar` is the option that lets us revert from the new (broken) urlbar design to the older one.
This setting should be renamed to something more like `zen.view.floating-urlbar` that would be true by default.
And this should be available as a setting on `about:preferences#zenLooks` page, as a simple boolean.

**Comment by RetrogradeDev** [comment]
There already is an urlbar option in the settings.

**Reply by Eragonfr** [reply]
Oh yup, I couldn't find it before. I guess I tried with an older version of zen.
